hotline/reports/forms.py

```diff
@@ -119,90 +107,83 @@ def iter_categories(self):
         for category in self.categories:
             yield category, self['category-%d' % category.pk]
 
-    def get_queryset(self):
-        # We do a monster join here so we get all the data we need on all the methods we
-        # call on the report
-        queryset = super().get_queryset().prefetch_related(
-            "image_set",
-            "comment_set__image_set"
-        ).select_related(
-            "reported_category",
-            "reported_species",
-            "actual_species",
-            "actual_species__severity",
-            "reported_species__severity",
-            "actual_species__category",
-            "reported_species__category"
-        )
+    def no_query_found(self):
+        """
+        If no results came up from a query, or the query is invalid, return all
+        of the reports
+        """
+        return self.searchqueryset.all().models(Report)
 
-        if not self.user.is_active:
-            # only show public reports, or reports that they were invited to,
-            # or reports that are in their session variable
-            queryset = queryset.filter(
-                Q(is_public=True) |
-                Q(pk__in=Invite.objects.filter(user_id=self.user.pk).values_list("report_id", flat=True)) |
-                Q(pk__in=self.report_ids)
-            )
+    def search(self):
+        """
```

Comment:
This docstring isn't necessary because it doesn't provide any information beyond what we already know about the `search` method.